components/icons/Logo.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
```

Comment:
karena ini file SVG ya, i think gak di taro di confignya chakra ui aja kah?

Reply:
Boleh2. I'll work on that shortly.

Reply:
@adibfirman Kayaknya agak susah deh. Soalnya logo React ID ini gak square, dan karena komponen `Icon` nge-force svg-nya jadi square, jadi mengecil di nav-nya

Reply:
i see, good to know mas, oke dari ku sisanya LGTM 💯